test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -33,6 +33,9 @@ Describe "TabCompletion" -Tags CI {
         $res.CompletionMatches[0].CompletionText | Should -BeExactly 'ToString('
```

Comment:
@daxian-dbw, your last commit had 3 failures in `PowerShell-CI-linux`
Built-in type information tests.Should have correct number of built-in type items in type table
```powershell
Expected exactly 272, but got 271.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/ETS/TypeTable.Tests.ps1: line 21
21:         $types.Count | Should -BeExactly ($IsWindows ? 273 : 272)
```
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @(@{InputObject=System.Diagnostics.TraceSource.dll; SideIndicator=<=}, @{InputObject=System.Runtime.Serialization.Primitives.dll; SideIndicator=<=}).
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/Startup.Tests.ps1: line 125
125:         $diffs | Should -BeExactly $null
```
Get-FormatData.Can get format data requiring '-PowerShellVersion 5.1'
```powershell
Expected a collection with size 5, but got collection with size 4 @(System.Management.Automation.FormatViewDefinition, System.Management.Automation.FormatViewDefinition, System.Management.Automation.FormatViewDefinition, System.Management.Automation.FormatViewDefinition).
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Get-FormatData.Tests.ps1: line 18
18:         $format.FormatViewDefinition | Should -HaveCount ($IsWindows ? 4 : 5)
```